docs/src/app/components/pages/discover-more/Showcase.jsx

```diff
@@ -42,6 +42,12 @@ const appList = [
     img: 'images/showcase/cloudcraft.png',
     link: 'https://cloudcraft.co/',
   },
+  {
+    title: 'It\'s quiz',
+    author: 'It\'s quiz',
+    img: 'images/showcase/itsquiz.png',
+    link: 'https://app.itsquiz.com/',
```

Comment:
Ahhh, made a typo, I've changed a link. Yes, we're going to support it in the nearest future.